exporter/loadbalancingexporter/metrics.go

```diff
@@ -18,18 +18,58 @@ import (
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
 	"go.opencensus.io/tag"
+	"go.opentelemetry.io/collector/obsreport"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/unit"
+	"go.opentelemetry.io/otel/sdk/metric/aggregation"
+	otelview "go.opentelemetry.io/otel/sdk/metric/view"
 )
 
 var (
 	mNumResolutions = stats.Int64("loadbalancer_num_resolutions", "Number of times the resolver triggered a new resolutions", stats.UnitDimensionless)
 	mNumBackends    = stats.Int64("loadbalancer_num_backends", "Current number of backends in use", stats.UnitDimensionless)
 	mBackendLatency = stats.Int64("loadbalancer_backend_latency", "Response latency in ms for the backends", stats.UnitMilliseconds)
 
+	meter = global.Meter("loadbalancingexporter")
+
+	omBackendLatency, _ = meter.SyncInt64().Histogram(
+		"loadbalancer_backend_latency",
+		instrument.WithDescription("Response latency in ms for the backends"),
+		instrument.WithUnit(unit.Milliseconds),
+	)
+
 	endpointTagKey      = tag.MustNewKey("endpoint")
 	successTrueMutator  = tag.Upsert(tag.MustNewKey("success"), "true")
 	successFalseMutator = tag.Upsert(tag.MustNewKey("success"), "false")
 )
 
+func init() {
```

Comment:
Looks good, thanks for confirming! I know this isn't a final code, but you might want to move this to the "Start" function or similar, wrapping it inside a sync.Once

Reply:
I think the views may need to be registered in an `init()` function as they need to be provided to the SDK before instruments are created.